MSFmultiSelect.js

```diff
@@ -90,13 +90,23 @@ class MSFmultiSelect {
     if (searchVal.length < 1) return;
 
     var i, optinVal, option, optionsLen = options.length;
+    var allSelected = true;
     for (i = 0; i < optionsLen; i++) {
       option = options[i];
       optinVal = option.innerText.toLocaleLowerCase();
 
       if (optinVal.indexOf(searchVal) !== 0) {
         option.parentElement.classList.add('hidden');
+        continue;
       }
+      allSelected &&= option.firstChild.checked;
+    }
+    self.uncheckSelectAllBtn(allSelected);
```

Comment:
Select All btn would be uncheck when all the searched labels are not checked. so user have a option to unselect all options immediately.